c10/core/Scalar.h

```diff
@@ -300,12 +300,13 @@ class C10_API Scalar {
   }
 
   Scalar(c10::SymBool sb) {
-    if (sb.is_symbolic()) {
+    // Why do we store bool in a double? (preexisting)
```

Comment:
Seems like a bug here. When we look at other code I see that bool is accessed via int field